.travis.yml

```diff
@@ -3,10 +3,11 @@ python:
     - "2.7"
 install:
     - pip install requests junit_xml beaker-client mock
-    - pip install pylint pycodestyle
+    - pip install pylint pycodestyle flake8
+before_script:
+    - flake8 --show-source .
```

Comment:
Hmm, this means that nothing from `script` would run, if this fails. Was that your intention?

Reply:
Is `script` a directory somewhere? I'm not sure where that might be.

Running `flake8` like this is recommended in their docs and it's a common practice when using `tox` for python testing, too.

Reply:
No, I meant `script` array next to `before_script` in `.travis.yml`. Travis CI doc [says](https://docs.travis-ci.com/user/customizing-the-build/#Breaking-the-Build) if any of the `before_script` commands fails, nothing under `script` runs, or so I understood. I'm not sure why flake8 recommends that, but perhaps because they think it's very important. I'd say being able to also get test results in a single CI run is a good thing, so I would move that to `script`.